src/Tes/Tes.csproj

```diff
@@ -12,8 +12,8 @@
 
   <ItemGroup>
     <PackageReference Include="EFCore.NamingConventions" Version="8.0.3" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.3" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="8.0.2">
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.4" />
```

Comment:
These will conflict with postgre and they need to stay at 8.0.2 until updates come from postgre